lib/bmt/version.rb

```diff
@@ -1,3 +1,3 @@
 module Bmt
-  VERSION = '0.3.0'.freeze
+  VERSION = '0.4.0'.freeze
```

Comment:
That checklist has "I have not incremented version.rb" - I think this doesn't work the way we think it should :D

Reply:
hahaha. was kind of wondering the reason of that :D